scripts/docker-entrypoint.sh

```diff
@@ -100,6 +101,11 @@ if [[ "$1" != "/"* ]]; then
         install_languages "$PAPERLESS_OCR_LANGUAGES"
     fi
 
+    if [ "$@" = "gunicorn" ]; then
+      cd /usr/src/paperless/src/ && \
+      exec sudo -HEu paperless GUNICORN_CMD_ARGS="--bind=0.0.0.0:8000 --access-logfile=- --log-file=-" /usr/bin/gunicorn paperless.wsgi
+    fi
```

Comment:
Even though I won't be needing it, I think that the args passed to gunicorn should be configurable. Maybe use something like the following instead?

```bash
if [[ "$1" = "gunicorn" ]]; then
    supplied_args="${@:2}"
    gunicorn_cmd_args="${supplied_args:---bind=0.0.0.0:8000 --access-logfile=- --log-files=-}"

    cd /usr/src/paperless/src/ && \
        exec sudo -HEu paperless GUNICORN_CMD_ARGS="$gunicorn_cmd_args" /usr/bin/gunicorn paperless.wsgi
else
    exec sudo -HEu paperless "/usr/src/paperless/src/manage.py" "$@"
fi
```

This will check if the first supplied parameter is `gunicorn`, and will pass the rest into `GUNICORN_CMD_ARGS` if specified, otherwise the defaults you had shown.

(I have also added the `else`-branch, because I find this more specific, although it doesn't change the behaviour at all.)